openapi/components/schemas/OrderSubscription.yaml

```
@@ -0,0 +1,281 @@
+required:
+  - type
+  - planId
+properties:
+  id:
+    type: string
+    description: ID of the order item.
+    readOnly: true
+    maxLength: 50
+    example: sub_01HRF27SATGE4Z6PBJE6PD8328
+  type:
+    description: Specifies the type of order.
+    type: string
+    x-basic: true
+    enum:
+      - subscription
+  renewalReminderTime:
+    description: Date and time when the renewal reminder event triggers.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  renewalReminderNumber:
+    description: Number of triggered renewal reminder events.
+    type:
+      - 'integer'
+      - 'null'
+    readOnly: true
+  trialReminderTime:
+    description: Date and time when the trial reminder event triggers.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
+  trialReminderNumber:
+    description: Number of triggered trial reminder events.
+    type:
+      - 'integer'
+      - 'null'
+    readOnly: true
+  status:
+    description: |-
+      Status of the subscription service.
+      A subscription starts in the `pending` status, and becomes `active` when the service period begins.
+    type: string
+    readOnly: true
+    enum:
+      - pending
+      - active
+      - abandoned
+      - canceled
+      - churned
+      - paused
+      - voided
+      - completed
+      - trial-ended
+  inTrial:
+    description: Specifies if the subscription is currently in a trial period.
+    type: boolean
+    readOnly: true
+  isTrialEnabled:
+    description: |-
+      Specifies if there is a trial for this subscription.
+      Plans without trial prices are free trials.
+    type: boolean
+  trialEndTime:
+    description: |-
+      Time and date when the trial ends.
+      If a trial is enabled on this subscription, a value must be provided.
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+  isTrialOnly:
+    description: |-
+      Specifies if a subscription ends after a trial period.
+      If this value is `true`, recurring settings are ignored.
+    type: boolean
+    default: false
+  isTrialConverted:
+    description: |-
+      Specifies if a subscription had trial conversion.
+      Trial conversion is when at least one successful payment is made on the subscription after a trial.
+    type: boolean
+    readOnly: true
+    example: true
+  recurringInterval:
+    type:
+      - 'object'
+      - 'null'
+    description: |-
+      Recurring interval to override plan settings.
+      To use plan settings, set this value to `null`.
+
+      To use multiple plans in one subscription,
+      all plans must have the same recurring period length.
+    example: null
+    properties:
+      periodAnchorInstruction:
+        $ref: ./ServicePeriodAnchorInstruction.yaml
+  endTime:
+    description: Date and time when the subscription ends.
+    x-sortable: true
+    type:
+      - 'string'
+      - 'null'
+    format: date-time
+    readOnly: true
```

```suggestion
  churnTime:
  description: Date and time when the subscription is deactivated.
    x-sortable: true
    type:
      - 'string'
      - 'null'
    format: date-time
    readOnly: true
```

See https://github.com/Rebilly/api-definitions/pull/1836

---

Please fix the description formatting